src/walletSdk/Anchor/index.ts

```diff
@@ -70,7 +70,7 @@ export class Anchor {
     }
 
     // fetch fresh TOML values from Anchor domain
-    const stellarToml = await StellarTomlResolver.resolve(this.homeDomain);
+    const stellarToml = await StellarToml.Resolver.resolve(this.homeDomain);
```

Comment:
👋 @Shaptic  btw the [migration guide](https://gist.github.com/Shaptic/5ce4f16d9cce7118f391fbde398c2f30) has `StellarSdk.Toml` instead of `StellarSdk.StellarTomlResolver`